docs/faq/lit-element-lifecycle.md

```diff
@@ -0,0 +1,82 @@
+# LitElement lifecycle
+
+## Overview
+
+Below is an overview of the most useful lifecycle callbacks. Check out the [official documentation](https://lit-element.polymer-project.org/guide/lifecycle) for a full overview.
+
+```js
+class MyElement extends LitElement {
+  /**
+   * Called once in the lifetime of an element, when it is instantiated.
+   * Useful for one-time synchronous setup work. You do not have access to
+   * any DOM in the constructor.
+   *
+   * You should not perform any side effects here, because someone might
+   * create your element without ever appending it to the DOM. Registering
+   * event listeners on your own element is fine.
+   */
+  constructor() {
+    super();
+  }
+
+  /**
+   * Called when the element is added to the page, useful for setup work which
+   * requires access to the DOM. However, your element has not done it's initial
+   * render here yet.
+   *
+   * Can be called multiple times when your element is moved around on the page,
+   * for example when using drag & drop.
+   *
+   * You might need to make sure your setup work runs only once, or you can clean
+   * it up in the disconnectedCallback.
+   */
+  connectedCallback() {
+    super.connectedCallback();
```

Comment:
Maybe a note on _why_ we have to call super.connectedcallback?

Reply:
Good point, added it